mobilecoind/src/config.rs

```diff
@@ -200,7 +200,8 @@ impl Config {
                 .build(),
         );
 
-        let conn = GrpcFogReportConnection::new(env, logger);
+        // TODO: Supply a network-id to mobilecoind?
```

Comment:
Looks like it is supplied?

Reply:
Will take a look and see what the status is in master

Reply:
i opened a PR in master to fix these